test/integration/features/stories/restart.feature

```diff
@@ -0,0 +1,82 @@
+@restart
+Feature: Restart
+Python developer checks out whether OpenShift is something she would like to use for her Django projects.
```

Comment:
@agajdosi The description does not match the issue description as IMO it is about the restart scenario which is applicable lot of things in Minishift. The original issue was https://github.com/minishift/minishift/issues/2679 which we had missed to catch in tests.